Tasks/Xcode/Tests/L0.ts

```diff
@@ -458,9 +458,94 @@ describe('Xcode L0 Suite', function () {
             '-exportPath /user/build/_XcodeTaskExport_testScheme -exportOptionsPlist _XcodeTaskExportOptions.plist'),
             'xcodebuild exportArchive should have been run with -allowProvisioningUpdates to export the IPA from the .xcarchive');
 
+        assert(tr.ran("/usr/libexec/PlistBuddy -c Add iCloudContainerEnvironment string Production _XcodeTaskExportOptions.plist") === false,
+            'PlistBuddy add cloud entitlement should not run');
+
+        assert(tr.stderr.length == 0, 'should not have written to stderr');
+        assert(tr.succeeded, 'task should have succeeded');
+        assert(tr.invokedToolCount == 21, 'Should have run \"PlistBuddy -c Add...\" four times, and 17 other command lines.');
+
+        done();
+    });
+
+    it('Xcode 9 signing defaults to manual, with auto export and cloud entitlement for production', (done: MochaDone) => {
+        this.timeout(1000);
+
+        const tp = path.join(__dirname, 'L0XCode9SigningDefaultsToAutoWithAutoExportAndCloudEntitlement.js');
```

Comment:
ToAuto or ToManual? Test title and filename disagree.

Reply:
How about `L0Xcode9ExportArchiveWithAutoAndCloudEntitlementForProduction`?